<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>digital_clock::gui::PluginListItemWidget</class>
 <widget class="QWidget" name="digital_clock::gui::PluginListItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>26</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0,0">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QCheckBox" name="name_check_box">
     <property name="text">
      <string notr="true">CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="version_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true">0.0.0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="config_btn">
     <property name="icon">
      <iconset resource="../resources/icons.qrc">
       <normaloff>:/clock/images/settings.svg.p</normaloff>:/clock/images/settings.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="info_btn">
     <property name="icon">
      <iconset resource="../resources/icons.qrc">
       <normaloff>:/clock/images/info.svg.p</normaloff>:/clock/images/info.svg.p</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
